sdk/tools/asmpp/asmpp.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Did you leave those in for easier debugging, or did you forget to delete em?

Reply:
Easier debugging. I guess it should work with CMaking a solution file as well. So if prefered, I can remove them.

Reply:
Up to you.